API/app.js

```diff
@@ -28,6 +34,20 @@ app.use((err, req, res, next) => {
   next(err);
 });
 
+app.use((err, req, res, next) => {
+  if (err.code === "23502") {
+    res.status(400).send({ msg: "Malformed Body" });
+  }
+  next(err);
+});
+
+app.use((err, req, res, next) => {
+  if (err.code === "23503") {
+    res.status(400).send({ msg: "User does not exist" });
+  }
+  next(err);
+});
```

Comment:
Great use of error handling middleware functions to catch a new PSQL error - you could combine these two functions into one with chained if statements that just handles PSQL errors
It would also be great if you could separate your error handling middleware functions into their own file to keep `app.js` tidy 🧹